docs/scripts_md/Private.md

```diff
@@ -0,0 +1,47 @@
+# NAME
+
+DICOM::Private -- Definitions of (optional) private fields of DICOM headers.
+
+# SYNOPSIS
+
+    use DICOM::Element;
+    use DICOM::Fields;    # Standard header definitions.
+    use DICOM::Private;   # Private or custom definitions.
+
+    # Initialize dictionary.
+    # Read the contents of the DICOM dictionary into a hash by group and element.
+    # dicom_private is read after dicom_fields, so overrides common fields.
+    BEGIN {
```

Comment:
If I understand correctly, in order to use the private fields one has to type this `BEGIN { .... }` block at the start of the script? If so, I think there are better strategies for implementing a DICOM dictionary. Let's leave it like this for now and come back at a later time.

Reply:
yeah. I found that a little weird. We can definitely improve this!